bin/gvs

```diff
@@ -95,30 +95,53 @@ for REPO in "${REPOSITORIES[@]}"; do
 
   # Checkout
   if $DO_CHECKOUT; then
-    # Delete composer dependencies from WooCommerce
-    if [ -f woocommerce.php ]; then
-      $VERBOSE_OUTPUT && echo "Delete composer packages from $(pwd)" || true
-      rm -f composer.lock && rm -rf vendor
-    fi
-
     # Checkout target
     git checkout --force --quiet "$REPO_TARGET"
 
-    # Install composer dependencies for WooCommerce
-    if [ -f woocommerce.php ]; then
-      ORIGINAL_VERSION=$(composer --version | cut -d " " -f 3)
-      if [ "${ORIGINAL_VERSION::1}" == "2" ]; then
-        echo "Switching composer version to install WooCommerce dependencies"
-        composer_self_update 1
-      fi
-      $VERBOSE_OUTPUT && echo "Install composer packages to $(pwd)" || true
-      composer install --no-suggest --no-dev --quiet
-      NEW_VERSION=$(composer --version | cut -d " " -f 3)
-      if [ "${ORIGINAL_VERSION::1}" != "${NEW_VERSION::1}" ]; then
-        echo "Switching composer version to install WooCommerce dependencies"
-        composer_self_update rollback
+    # Directories to look for a composer.json file in
+    declare -a dirs=("$REPO_PATH" "$REPO_PATH/plugins/woocommerce")
```

Comment:
```suggestion
    DIRS=("$REPO_PATH" "$REPO_PATH/plugins/woocommerce")
```

The `declare` part is not necessary.

Please use all caps for variable names to match existing code (need more editing; I did not update everywhere).